compiler/hash-attrs/src/builtin.rs

```diff
@@ -30,7 +30,7 @@ macro_rules! define_attr {
     ($table:expr, $name:ident, { ($($arg:ident : $ty:ident),*), $subject:expr }) => {
         let name: Identifier = stringify!($name).into();
 
-        let params = if ${count(arg)} == 0 {
+        let params = if ${count($arg)} == 0 {
```

Comment:
Should `stringify!(arg)` below also be `$arg`?

Reply:
This was just an issue that came up in the new Nightly release which complained about it, I don't think we need to change anything else. From what I understand, it is a metavariable expression issue.